website/server/libs/inbox/index.js

```diff
@@ -74,7 +74,41 @@ export async function getUserInbox (user, options = {asArray: true, page: 0, con
   }
 }
 
+async function usersMapByConversations (owner, users) {
+  let query = Inbox
+    .aggregate([
+      {
+        $match: {
+          ownerId: owner._id,
+          uuid: { $in: users },
+        },
+      },
+      {
+        $group: {
+          _id: '$uuid',
+          user: {$last: '$user' },
+          username: {$last: '$username' },
+          userStyles: {$last: '$userStyles'},
+          contributor: {$last: '$contributor'},
+
+          count: {$sum: 1},
```

Comment:
is this used anywhere?

Reply:
no, not in the 2nd query - I removed the duplicated properties of the 2nd query